flink-connectors/flink-connector-aws-kinesis-firehose/src/test/java/org/apache/flink/connector/firehose/sink/KinesisFirehoseSinkWriterTest.java

```diff
@@ -74,30 +73,28 @@ public void getSizeInBytesReturnsSizeOfBlobBeforeBase64Encoding() {
                 .isEqualTo(testString.getBytes(StandardCharsets.US_ASCII).length);
     }
 
-    @Test
+    @Test(expected = KinesisFirehoseException.KinesisFirehoseFailFastException.class)
```

Comment:
If we expect to throw why is there an assertion on the last line of the test? We should be using assertJ for exception checking `Assertions.assertThatExceptionOfType(..)`